### ISSUE TITLE
Provide sane way of db extractor setup even if `getTables` api call exceeding timeout

### ISSUE THREAD

**Issue by ujovlado** [keboola/kbc-ui#1619]
Mame situaciu, ze sync api call na list tables nedobehne do pozadovaneho limitu.

Extractor sa da normalne nakonfigurovat (v Advanced Mode) pomocou manualneho pisania queries.

Avsak vzdy v detaile konfiguracie sa opat robi ten check, ktory nezbehne.

Priklad:
https://keboola.zendesk.com/agent/tickets/8213

Riesenie:

- pridat lightweight sync akciu, ktora spravi LEN getTables bez udajov k stlpcom
- pripadne v UI zabezpecit aby to nerobilo stale ten check

cc: @kacurez @pivnicek

**Comment by kacurez** [comment]
imo to getTables bez stlpcov je najlepsie riesenie. Asi to nemusi byt novy sync api call ale parametrizovany nieco ako getTables(includeColumns=false). Dalej ma napada ma ze by sa tam potom na ziskanie stlpcov pre konkretnu tabulku zavolal sync api call az po vybrati tabulky a ui by si vysledky pamatalo.

**Comment by pivnicek** [comment]
jo, I think that maybe it could be best to have 2 separate synch calls.  1) Get tables (without columns as @kacurez said, can use the same action with an additional parameter), 2) Get columns (for a single table).
So when the user selects the table, it will call the second action and load up the `columns` / `primaryKey` etc elements.
That could maybe work.
